package.json

```diff
@@ -28,27 +27,21 @@
   "devDependencies": {
     "bower": "^1.8.2",
     "broccoli-asset-rev": "^2.4.5",
-    "ember-ajax": "^2.4.1",
+    "ember-ajax": "^2.5.6",
     "ember-cli": "2.12.3",
-    "ember-cli-chai": "^0.4.3",
     "ember-cli-dependency-checker": "^1.3.0",
     "ember-cli-frost-blueprints": "^4.0.0",
     "ember-cli-htmlbars": "^1.1.1",
-    "ember-cli-htmlbars-inline-precompile": "^0.3.6",
+    "ember-cli-htmlbars-inline-precompile": "0.3.12",
     "ember-cli-inject-live-reload": "^1.4.1",
-    "ember-cli-mocha": "^0.14.4",
-    "ember-cli-release": "^0.2.9",
     "ember-cli-shims": "^1.0.2",
-    "ember-cli-sri": "^2.1.0",
     "ember-cli-uglify": "^1.2.0",
     "ember-disable-prototype-extensions": "^1.1.0",
     "ember-export-application-global": "^1.0.5",
-    "ember-hook": "^1.4.2",
     "ember-load-initializers": "^0.6.0",
     "ember-resolver": "^2.0.3",
-    "ember-sinon": "^0.7.0",
     "ember-source": "~2.12.0",
-    "ember-test-utils": "^7.0.1",
+    "ember-test-utils": "^8.0.0",
```

Comment:
Do we have any content in this repo that benefits from the linting? Guess there's some files...

Reply:
Yes, we need this to provide linting via `ember run test`